packages/core/src/browser/shell/application-shell.ts

```diff
@@ -219,6 +220,7 @@ export class ApplicationShell extends Widget {
         @inject(ApplicationShellOptions) @optional() options: RecursivePartial<ApplicationShell.Options> = {},
         @inject(CorePreferences) protected readonly corePreferences: CorePreferences,
         @inject(SaveResourceService) protected readonly saveResourceService: SaveResourceService,
+        @inject(SecondaryWindowHandler) protected readonly secondaryWindowHandler: SecondaryWindowHandler,
```

Comment:
I believe this is a breaking change that should be documented:
- https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#property-injection